scss/components/_accordion.scss

```diff
@@ -14,6 +14,14 @@ $accordion-background: $white !default;
 /// @type Boolean
 $accordion-plusminus: true !default;
 
+/// Default indicator for plus when $accordion-plusminus is true
+/// @type String
+$accordion-plus-character: '\002B' !default;
```

Comment:
Could you rename it to `$accordion-plus-content` as this is a String that can contain several characters ?